dev_guide/new_app.adoc

```diff
@@ -70,3 +70,42 @@ values defined by the template:
 $ osc new-app ruby-helloworld-sample -p ADMIN_USERNAME=myuser,ADMIN_PASSWORD=mypass
 ----
 
+== Using the Management Console
```

Comment:
Given this new heading, the above "Usage Examples" could probably become "Using the CLI".